index.html

```diff
@@ -39,6 +39,190 @@ <h2 class="connect">
+                </ul>
+                <p class="canopy-p">
+                    A daily section of privately personalized reads: no accounts or signups required.
+                </p>
+                <div class="lang-div">
+                    <ul class="language">
+                        <li class="button-html">Html</li>
+                        <li class="button-css">CSS</li>
+                        <li class="button-js">Javascript</li>
+                      </ul>
+                </div>
+                
+                <div>
+                    <button class="projects-button" type="submit">See Projects</button>
+                </div>
+            </div>
+
+
+        </div>
+
+        <!-- About Page -->
+
+        <section class="about-container">
+            <!-- <header>
+                <nav class="navbar">
+                        <a href="#" class="logo-link">My Logo</a>
+                    <div class="menu">
+                        <i class="fas fa-bars"></i>
+                    </div>
+                </nav>     
+            </header> -->
+            <section class="intro">
+                <h1 class="hero">About <br>
+                    Myself
+                </h1>
+                <p class="message">Thank you for stopping by. Please feel free to browse my portfolio. I'm a software developer and love creating interesting things. If you have an idea that you need help with. I'll be happy to help bring it to life! Reach out anytime!</p>
+                <h2 class="connect"> 
+                    <a href="#">Reach Out</a>
+                </h2>
+                <ul class="connent-icons">
+                    <li><a href="#"></a><img src="images/twitter.png" alt="twitter"></li>
+                    <li><a href="https://github.com/gracekabaghe"></a><img src="images/github.png" alt="github"></li>
+                    <li><a href="#"></a><img src="images/linkedin.png" alt="linkedin"></li>
+                    <li><a href="#"></a><img src="images/medium.png" alt="medium"></li>
+                </ul>
+
+                <div class="connect-button">
+                    <button class="about-button" type="submit">Get my resume</button>
+                </div>
+
+                <h2>Languages</h2>
+
+                <div class="html-shape">
+                    <img src="images/jsimg.png" alt="javascript">
+                    <p>Javascript</p>
+                </div>
+                <div class="html-shape">
+                    <img src="images/html.png" alt="">
+                    <p>HTML</p>
+                </div>
+                <div class="html-shape">
+                    <img src="images/cssimage.png" alt="">
+                    <p>CSS</p>
+                </div>
+
+                <div class="html-shape1">
+                    <p>Frameworks</p>
+                </div>
+
+                <div class="html-shape1">
+                    <p>Skills</p>
+                </div>
```

Comment:
- [ ] Kindly modify this part as requested in the project requirements : 
![image](https://user-images.githubusercontent.com/43698511/133910663-8c494008-97b2-4fcc-ac1f-7ce8ad56cd28.png)
languages, frameworks, and skills should be wrapped in a `ul` tag and each technology should be wrapped in a `li` tag.